README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+---
+#"Computing for Social Sciences HW01"
+"**Taylor Griffin**"  
+"*October 8, 2019*"  
+
+# Introduction
+
+![Taylor Griffin](C:/Users/Taylor Griffin/Desktop/Computing R/Taylor G hw01/Taylor.png)
```

Comment:
It seems that I am not able to view your picture. I guess the reason might be that it exists only as a document on your PC.